Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1323,8 +1323,13 @@
 )
 #endif
 
+#if PLATFORM(MAC)
 (deny mach-lookup (with no-report)
     (global-name "com.apple.tccd.system"))
+#else
```

Comment:
I think it might be a bit clearer to state that we need this for Catalyst, so we don't forget.

Reply:
Ah, good point! I have a follow-up patch that should remove the need for this service on Catalyst, too. Will upload shortly.

Thanks for reviewing!